### PR TITLE
rust: update to 1.30.1.

### PR DESCRIPTION
**Pull request by Cogitri**
[ci skip]

### CODE REVIEW

**Comment by Cogitri** [comment]
Tested on x86_64 and aarch64-musl as of now

**Comment by jnbr** [comment]
checksums for `cargo-0.31.0-*-unknown-linux-gnu.tar.xz` changed

**Comment by Gottox** [comment]
musl bootstrap compiler updated:

https://alpha.de.repo.voidlinux.org/distfiles/rustc-1.30.1-x86_64-unknown-linux-musl.tar.xz
https://alpha.de.repo.voidlinux.org/distfiles/rust-std-1.30.1-x86_64-unknown-linux-musl.tar.xz

File: srcpkgs/rust/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'rust'
 pkgname=rust
-version=1.30.0
+version=1.30.1
 revision=1
 _rust_dist_version=1.29.2
```
Comment (Gottox) [inline srcpkgs/rust/template]: Update the rust_version to the new versions.

**Comment by Cogitri** [comment]
> checksums for cargo-0.31.0-*-unknown-linux-gnu.tar.xz changed

Uh? Seems to me as if they're the same

**Comment by jnbr** [comment]
If you still have the old file in `hostdir/sources`, `xbps-src` will not load it again and the old checksum still matches.

**Comment by Cogitri** [comment]
> If you still have the old file in hostdir/sources

Already tested, both `xgensum -f` and manually purging the files didn't change the result: it seems to me as if the checksum have stayed the same

**Comment by jnbr** [comment]
Well, maybe something is wrong on my site. No idea what it could be though.

**Comment by Cogitri** [comment]
Had something weird like that happen to me too the last time I've bumped rust, maybe smth about rust's download thingie is weird. Will double-check on another machine though

**Comment by Gottox** [comment]
Currently testbuilding the -musl platforms, glibc afterwards.

**Comment by Cogitri** [comment]
I was talking about:
> checksums for cargo-0.31.0-*-unknown-linux-gnu.tar.xz changed

**Comment by Gottox** [comment]
They have changed indeed.

**Comment by Cogitri** [comment]
Weird, it worked on two machines of mine. Mind sending me the new shasums?

**Comment by Gottox** [comment]
```
=> ERROR: SHA256 mismatch for 'cargo-0.31.0-x86_64-unknown-linux-gnu.tar.xz:'
21ef918ba8970815856ef2764ad1b377a2d4868bed96340ac05d7fe62f8cf952
```

```
=> ERROR: SHA256 mismatch for 'cargo-0.31.0-i686-unknown-linux-gnu.tar.xz:'
642e298e1431a9d65559a3bae9c4e1dab79816577b7b3c1f35e28c5621878165
```

I think that we will hit these kind of errors again, as it seems that github has changed the gzip algorithm

**Comment by jnbr** [comment]
merged

**Comment by Gottox** [comment]
merged

**Pull request closed without merging**